### PR TITLE
Support VanillaFoodPantry recipes.

### PR DESCRIPTION
**Pull request by gchpaco**
This adds a module for supporting VanillaFoodPantry, which makes extensive use of non-food intermediate recipes.

### CODE REVIEW

File: src/main/java/net/blay09/mods/cookingforblockheads/compat/VanillaFoodPantryAddon.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+    private static final String FRESH_WATER_ITEM = "water_portion";
+    private static final String FRESH_MILK_ITEM = "milk_portion";
+
+
+    public VanillaFoodPantryAddon() {
+        super("vanillafoodpantry");
+
+        addNonFoodRecipe(ADDITIONAL_RECIPES);
+        addTool(TOOLS);
```
Comment (BlayTheNinth) [inline src/main/java/net/blay09/mods/cookingforblockheads/compat/VanillaFoodPantryAddon.java]: `addWildcardNonFoodRecipe` doesn't seem to be used right now?

Comment (gchpaco) [reply src/main/java/net/blay09/mods/cookingforblockheads/compat/VanillaFoodPantryAddon.java]: Derp, that's wrong.

**Comment by gchpaco** [comment]
Rebased onto 1.11 with the width patch.

**Comment by BlayTheNinth** [comment]
Thank youuu

**Pull request merged**